zone.tf

```diff
@@ -20,27 +11,12 @@ resource "cloudflare_worker_route" "fairlane" {
   zone_id     = var.cloudflare_zone_id
   pattern     = "${var.domain}/*"
   script_name = cloudflare_worker_script.fairlane_worker.name
+  count       = var.create_worker_routes ? 1 : 0
```

Comment:
Place counts on the top of arguments. Saves time "parsing" the code while reading :)